app/ui.js

```diff
@@ -98,6 +98,8 @@ var UI;
         lastKeyboardinput: null,
         defaultKeyboardinputLen: 100,
 
+        reconnect_enabled: false,
```

Comment:
This is easily confused with the primary reconnect setting. Another name would be nice.